projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+.retry:
+  retry_options:
+    max: 1
+    when:
+      - always
+
+
+build-vdk-jupyterlab-extension:
+  stage: build
+  extends: .build-plugin-dind
+  image: "python:3.8-alpine"
```

Comment:
what image do the rest of the VDK gitlab ci jobs leverage for running python workloads?

Reply:
@ivakoleva I used that image to use apk for adding npm